packages/mongodb-memory-server-core/src/util/MongoBinary.ts

```diff
@@ -32,54 +32,65 @@ export interface MongoBinaryOpts {
 export default class MongoBinary {
   static cache: MongoBinaryCache = {};
 
+  /**
+   * Probe if the provided "systemBinary" is an existing path
+   * @param systemBinary The Path to probe for an System-Binary
+   * @return System Binary path or empty string
+   */
   static async getSystemPath(systemBinary: string): Promise<string> {
     let binaryPath = '';
 
     try {
       await promisify(fs.access)(systemBinary);
 
-      log(`MongoBinary: found sytem binary path at ${systemBinary}`);
+      log(`MongoBinary: found sytem binary path at "${systemBinary}"`);
       binaryPath = systemBinary;
     } catch (err) {
-      log(`MongoBinary: can't find system binary at ${systemBinary}. ${err.message}`);
+      log(`MongoBinary: can't find system binary at "${systemBinary}".\n${err.message}`);
     }
 
     return binaryPath;
   }
 
-  static async getCachePath(version: string): Promise<string> {
+  /**
+   * Check if specified version already exists in the cache
+   * @param version The Version to check for
+   */
+  static getCachePath(version: string): string {
     return this.cache[version];
   }
 
+  /**
+   * Probe download path and download the binary
+   * @param options Options Configuring which binary to download and to which path
+   * @returns The BinaryPath the binary has been downloaded to
+   */
   static async getDownloadPath(options: Required<MongoBinaryOpts>): Promise<string> {
     const { downloadDir, platform, arch, version, checkMD5 } = options;
-    // create downloadDir if not exists
+    // create downloadDir
     await mkdirp(downloadDir);
 
+    /** Lockfile path */
     const lockfile = path.resolve(downloadDir, `${version}.lock`);
-    // wait lock
+    // wait to get a lock
     await new Promise((resolve, reject) => {
       LockFile.lock(
         lockfile,
         {
-          wait: 120000,
+          wait: 1000 * 12, // 120 seconds
           pollPeriod: 100,
-          stale: 110000,
+          stale: 1000 * 11, // 110 seconds
```

Comment:
changed these numbers to be something more understandable

Reply:
I'll fix it 12 -> 120 in next commit